src/views/Correction.vue

```diff
@@ -363,6 +365,8 @@ export default class Correction extends Mixins(
     // this is the id of the original filing to correct
     this.correctedFilingId = +this.$route.params.correctedFilingId // number (may be NaN)
 
+    // this is the correction type of either client or staff
+    this.correctionUserType = this.$route.params.correctionUserType
```

Comment:
Do you have a clear direction that changing this internal correction is in scope?

I think changing internal correction filings should be out of scope for now, especially since these will be changed to "external" (Edit UI) filings soon.

Reply:
Okay 👌